### PR TITLE
Add MouseFix v0.9.0 (initial release)

### PR DESCRIPTION
**Pull request by josser**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
There is no stable but 0.9.0 is the only and initial version
Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].
- [x] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

File: Casks/mousefix.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'mousefix' do
+  version '0.9.0'
+  sha256 'f6290b3251f9152246b8d06ad8bf8249976fe0f69b09cb031fedbe6b3e8c25c9'
+
+  url 'https://mousefix.org/maindownload/MacMouseFix.zip'
+  name 'Mouse Fix'
+  homepage 'https://mousefix.org/'
+
+  auto_updates true
+  container type: :zip
```
Comment (vitorgalvao) [inline Casks/mousefix.rb]: Are you sure this is necessary? It should work without it.

File: Casks/mousefix.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'mousefix' do
+  version '0.9.0'
```
Comment (vitorgalvao) [inline Casks/mousefix.rb]: When a `url` doesn’t point to a specific version and there’s no [appcast](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md), a cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/homebrew/homebrew-cask/blob/37f8d181d1789cbb06c6e73625587fdbe77ca225/Casks/kibako.rb#L2L3).

File: Casks/mousefix.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'f6290b3251f9152246b8d06ad8bf8249976fe0f69b09cb031fedbe6b3e8c25c9'
+
+  url 'https://mousefix.org/maindownload/MacMouseFix.zip'
+  name 'Mouse Fix'
+  homepage 'https://mousefix.org/'
+
+  auto_updates true
```
Comment (vitorgalvao) [inline Casks/mousefix.rb]: For this to be true, an appcast needs to exist. A possible one is `https://mousefix.org/maindownload/bundleversion`.

File: Casks/mousefix.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+cask 'mousefix' do
```
Comment (vitorgalvao) [inline Casks/mousefix.rb]: `mouse-fix`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> + The Cask is too obscure. Examples:
>   + A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 50 stars).
>   + [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021).

**Comment by josser** [comment]
So, I'll wait until app gain more popularity and will fix the issues :D 
btw, It's really cool, and got a lot of points on reddit: https://www.reddit.com/r/mac/comments/dca249/i_made_mac_mouse_fix_a_simple_mac_app_that_will/

**Comment by ran-dall** [comment]
@josser I have also archived a fixed copy in my repo. https://github.com/ran-dall/homebrew-repo/commit/a63551a34c11b6a2c91a00b14a4376d20448a72f

**Pull request closed without merging**